app/models/shipit/user.rb

```diff
@@ -8,6 +8,8 @@ class User < ActiveRecord::Base
     has_many :commits, foreign_key: :committer_id, inverse_of: :committer
     has_many :tasks
 
+    attr_encrypted :access_token, key: Shipit.user_access_tokens_key
```

Comment:
I don't it's necessary to prefix with `github_` here because the `User` model is literally github users.